src/PyCall.jl

```diff
@@ -286,6 +286,20 @@ end
 
 getindex(o::PyObject, s::Symbol) = convert(PyAny, getindex(o, string(s)))
 
+function getproperty(o::PyObject, s::Symbol)
+    if s == :o # we still want to be able to access o.o
+        getfield(o, s)
+    else
+        getindex(o, s)
+    end
+end
+
+function propertynames(o::PyObject)
+    members = convert(Vector{Tuple{AbstractString,PyObject}},
```

Comment:
I don't think we need the `convert` call here since the `getmembers` result is iterable.  We can just do `map` on the result.